app/_src/gateway/breaking-changes/37x.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Kong Gateway 3.7.x breaking changes
+content_type: reference
+book: breaking-changes
+chapter: 9
+---
+
+Before upgrading, review any configuration or breaking changes in this version and prior versions that
+affect your current installation.
+
+You may need to adopt different [upgrade paths](/gateway/{{page.release}}/upgrade/#guaranteed-upgrade-paths) depending on your 
+deployment methods, set of features in use, or custom plugins, for example.
+
+Review the [changelog](/gateway/changelog/#3700) for all the changes in this release.
+
+## Breaking changes and deprecations
+
+### Configuration
+
+The Granular Tracing feature has been removed from {{site.base_gateway}}.
```

Comment:
```suggestion
The Granular Tracing feature has been deprecated and removed from {{site.base_gateway}}.
```

Reply:
I'm not sure if it was actually deprecated but listing it as 'removed' left me wanting more info

Reply:
It's both. It was deprecated in 3.0: https://docs.konghq.com/gateway/changelog/#breaking-changes-and-deprecations-5:~:text=Support%20for%20the%20nginx%2Dopentracing%20module%20is%20deprecated%20as%20of%203.0%20and%20will%20be%20removed%20from%20Kong%20in%204.0%20(see%20the%20Known%20Limitations%20section%20for%20additional%20information).

It's now fully removed (was supposed to be in 4.0, but seems like they did it earlier).